lib/h264dec.cpp

```diff
@@ -19,7 +19,6 @@
 
 #include <atomic>
 
-#include "base.h"
```

Comment:
Ouch... it would be possible to split this also? a commit related with importing headers, and other with h265 parsing?